engine/src/CalciteInterpreter.cpp

```diff
@@ -431,7 +148,6 @@ std::unique_ptr<ral::frame::BlazingTable> execute_plan(std::vector<ral::io::data
 									"substep"_a=queryContext.getQuerySubstep(),
 									"info"_a="In execute_plan. What: {}"_format(e.what()),
 									"duration"_a="");
-		logger->flush();
```

Comment:
porque borraste ese flush? la idea es que cuando hay un error, se le haga flush, para asegurar que se escribe al archivo antes de que crashee el proceso

Reply:
I'm calling `spdlog::flush_on(spdlog::level::err);` in `initialize.cpp` which does exactly the same